cmd/2019_3/main.go

```diff
@@ -33,24 +29,16 @@ func init() {
 	utils.Init(&flagInput)
 }
 
-func abs(x int) int {
-	if x < 0 {
-		return -x
-	}
-
-	return x
-}
-
 func newGrid(wirePaths []string) *grid {
 	g := new(grid)
-	g.grid = make(map[point][]wire)
+	g.grid = make(map[point.Point][]wire, len(wirePaths))
 
 	for index, wirePath := range wirePaths {
 		if wirePath == "" {
 			continue
 		}
 
-		g.cross = []point{}
+		g.cross = make([]point.Point, 0, len(wirePaths))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '0', create a named constant for it](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498955)